src/System.Runtime.InteropServices.RuntimeInformation/src/project.json

```diff
@@ -6,10 +6,12 @@
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
-    "System.Threading": "4.0.0"
+    "System.Threading": "4.0.0",
+    "System.Linq": "4.0.0",
```

Comment:
Neither of these should be needed once you switch to more efficient method to get the custom attribute.